### PR TITLE
Wip 178185316 building attributes

### PR DESCRIPTION
**Pull request by kwiliarty**

### CODE REVIEW

**Comment by kwiliarty** [comment]
I haven't yet torn out the old 'Unit attributes.' Wanted to check before doing that...

File: app/controllers/housing_attributes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+class HousingAttributesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_housingable # Must be defined on the child class
+  before_action :set_attribute, only: [:edit, :update, 'destroy']
```
Comment (eanders) [inline app/controllers/housing_attributes_controller.rb]: `:destroy` can be a symbol.

File: app/views/housing_attributes/_edit.haml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+-#   Arguments:
+-#     housingable: currently a building or unit object
+-#     attribute: a HousingAttribute object
+
+= content_for :modal_title do
+  Edit Attribute
+
+= simple_form_for @attribute, url:  polymorphic_path([@housingable, @attribute]) do |f|
+  = f.input :name, disabled: true
+  = f.input :value, collection: @attribute.existing_values, input_html: { class: :select2, data: { tags: true }}, include_blank: true, hint: 'If you don\'t see the value required, type it in and select it'
```
Comment (eanders) [inline app/views/housing_attributes/_edit.haml]: I believe if you use `as: :select_2` then you don't have to add the class or the javascript at the bottom of the partial.

File: app/controllers/housing_attributes_controller.rb (hunk 1 above)
Comment (kwiliarty) [reply app/controllers/housing_attributes_controller.rb]: 👍

File: app/views/housing_attributes/_edit.haml (hunk 2 above)
Comment (kwiliarty) [reply app/views/housing_attributes/_edit.haml]: Yes, thanks for that.

**Comment by kwiliarty** [comment]
I've ripped out the db structure and the code that was in place for `UnitAttributes`

**Pull request merged**